Update Home text (#11319)
home-assistant/frontend: gallery/src/pages/concepts/home.markdown

# Welcome to Home Assistant Design

This portal aims to aid designers and developers on improving the Home Assistant interface. It consists of working code, resources and guidelines.

## Home Assistant interface

> The Home Assistant frontend allows users to browse and control the state of their house, manage their automations and configure integrations. The frontend is designed as a mobile-first experience. It is a progressive web application and offers an app-like experience to our users. The Home Assistant frontend needs to be fast. But it also needs to work on a wide range of old devices.

Reviewer: Not every home is a house :)

```suggestion
The Home Assistant frontend allows users to browse and control the state of their home, manage their automations and configure integrations. The frontend is designed as a mobile-first experience. It is a progressive web application and offers an app-like experience to our users. The Home Assistant frontend needs to be fast. But it also needs to work on a wide range of old devices.
```

### Material Design

> The Home Assistant interface is based on Material Design. It's a design system created by Google to quickly build high-quality digital experiences. Components and guidelines that are custom made for Home Assistant are documented on this portal. For all other components please go to <a href="https://material.io" rel="noopener noreferrer target="_blank">material.io</a>.

Reviewer 2: ```suggestion
The Home Assistant interface is based on Material Design. It's a design system created by Google to quickly build high-quality digital experiences. Components and guidelines that are custom made for Home Assistant are documented on this portal. For all other components please go to <a href="https://material.io" rel="noopener noreferrer" target="_blank">material.io</a>.
```

> The Home Assistant interface is based on Material Design. It's a design system created by Google to quickly build high-quality digital experiences. Components and guidelines that are custom made for Home Assistant are documented on this portal. For all other components please go to <a href="https://material.io" rel="noopener noreferrer" target="_blank">material.io</a>.

Reviewer: ```suggestion
The Home Assistant interface is based on Material Design. It's a design system created by Google to quickly build high-quality digital experiences. Components and guidelines that are custom made for Home Assistant are documented on this portal. For all other components check <a href="https://material.io" rel="noopener noreferrer" target="_blank">material.io</a>.
```
